spec/spec_helper.rb

```diff
@@ -63,4 +62,15 @@
   config.filter_run_excluding :broken => true
 
   config.fail_fast = false
+
+  config.before(:all) {
+    if self.class.metadata[:vcr]
+      VCR.insert_cassette("#{self.class.metadata[:description]}/all")
+    end
+  }
+  config.after(:all) {
+    if self.class.metadata[:vcr]
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.